src/constants/pools.ts

```diff
@@ -406,6 +412,28 @@ const POOLS_MAINNET: Pools = {
     '0x5b3240b6be3e7487d61cd1afdfc7fe4fa1d81e6400000000000000000000037b',
     '0x851523a36690bf267bbfec389c823072d82921a90002000000000000000001ed',
   ],
+  Deprecated: {
+    '0x8e85e97ed19c0fa13b2549309965291fbbc0048b0000000000000000000003ba': {
+      newPool: '',
+      suggestedPools: [],
+    },
```

Comment:
Because these are optional, if there are none don't set them:
```suggestion
    '0x8e85e97ed19c0fa13b2549309965291fbbc0048b0000000000000000000003ba': {},
```

Reply:
ah, I thought we wait for information about that new and suggested pools and will add in this pr, that's why I added it

ok, will remove